plugins/phishtank/plugin.spec.yaml

```diff
@@ -25,58 +45,74 @@ connection:
     title: API Key
     description: The API key to use
     type: credential_secret_key
-    required: true
+    required: false
+    example: secret_key
+  username:
+    title: Username
+    description: Phishtank username
+    type: string
+    required: false
+    example: UserExample
 actions:
   check:
     title: Check URL
-    description: Submit URL to PhishTank
+    description: Submit a URL to PhishTank
     input:
       url:
         title: URL
         description: URL to Submit
         type: string
         required: true
+        example: example.com
     output:
```

Comment:
Can we add all outputs returned from the API and API example outputs from the Phishtank website? `verification_time` is one but there may be more.